modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -595,24 +599,31 @@ private boolean _isFieldEmpty(
 			DDMFormFieldValue localizedObjectFieldDDMFormFieldValue =
 				new DDMFormFieldValue();
 
+			localizedObjectFieldDDMFormFieldValue.setDDMFormValues(
+				new DDMFormValues(ddmFormField.getDDMForm()));
 			localizedObjectFieldDDMFormFieldValue.setName(
 				ddmFormFieldValue.getName());
 
 			String defaultLanguageId = null;
+			String defaultLocale = String.valueOf(
+				ddmFormField.getProperty("defaultLocale"));
```

Comment:
I saw that Brian complained about this name. I though about renaming it to `defaultLocaleId` . What do you think?

Reply:
Hey @marcelabc,
I don't need to do it anymore because I added this [commit](https://github.com/liferay-bpm/liferay-portal/pull/4309/commits/c8c9f855c0641f5ac29f5158a6c6a6beaa8100bb)